tests/unit/wallet/test_wallet.py

```diff
@@ -17,6 +20,11 @@ async def asyncSetUp(self):
         self.main_ledger = self.manager.get_or_create_ledger(Ledger.get_id(), config)
         self.test_ledger = self.manager.get_or_create_ledger(RegTestLedger.get_id(), config)
 
+        base_dir = os.path.join(os.path.dirname(__file__), '../../../')
+        types_dir = os.path.join(base_dir, 'lbry/schema/types/v2')
```

Comment:
I know this is ugly but I couldn't find an example of doing anything similar in the codebase.